src/latis-python-client/latis.py

```diff
@@ -175,19 +347,36 @@ def getFile(self, filename, format='csv'):
             f.write(csv)
 
     def clearProjections(self):
+        """
+        Clears projections list.
+        """
         self.projections = []
 
     def clearSelections(self):
+        """
+        Clears selections list.
+        """
         self.selections = []
 
     def clearOperations(self):
+        """
+        Clears operations list.
+        """
         self.operations = []
 
 
 class Metadata:
+    """A class that represents metadata for a dataset.
 
-    def __init__(self, latisInstance, dataset):
+    Attributes:
+        properties (dict): The metadata properties of the dataset.
 
+    Args:
+        latisInstance (LatisInstance): The Latis instance.
+        dataset (Dataset): The dataset.
+    """
+
+    def __init__(self, latisInstance, dataset):
```

Comment:
In the `__init__` method, `latisInstance` and `dataset` could be explained in more detail. These arguments are critical to understanding how to use the class properly. Here's how I would revise the docstrings:

```
def __init__(self, latisInstance, dataset):
        """Initializes the Metadata object and retrieves the metadata properties.

        Retrieves metadata from the provided Latis instance and dataset. The
        retrieval method depends on the version of Latis being used.

        Args:
            latisInstance (LatisInstance): The Latis instance from which to
            retrieve metadata.
            dataset (Dataset): The dataset for which to retrieve metadata.
        """
```